content/english/chatbot/activity-2.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Activity 2: Setup PandoraBots"
+draft: false
+weight: 5
+---
+
+## AIML
+We will be using AIML to create our chatbot. AIML stands for **Artificial Intelligence Markup Language**. If you have ever used HTML to make a website, you will find that AIML looks very similar. Before we can start writing AIML, we need to sign up for an account on PandoraBots.
+
+## Pandorabots
+Go to [https://www.pandorabots.com/](https://www.pandorabots.com/) and click on **Sign Up**. Enter an email and password to create an account. Alternatevely, you can sign up using an account from Google, Facebook, Twitter, or Github.
+
+## Create a new bot
+Once you have signed up, you should see a page that looks like this:
+
+<p style="text-align: center;"><img src="../img/pandorabots_1.png" alt="new account landing page on pandorabots" width="1000"/></p>
+
+To create a new bot, use the “+” button next to My Bots on the navigation pane. Give it a name and click “Create Bot”.
+
+<p style="text-align: center;"><img src="../img/pandorabots_2.png" alt="creating a new bot on pandorabots" width="1000"/></p>
+
+## Bot Editing
+Once you have created a new bot, the name of the bot will appear in the navigation pane. Click **Edit** beneath your Bot’s Name to navigate to the Editor.

```

Comment:
Under Editor, maybe specify the user has to select Code Editor to get to the screen you have in the screenshot.